src/containers/course-sections-list/CourseSectionsList.tsx

```diff
@@ -31,16 +38,31 @@ interface CourseSectionsListProps {
     value: CourseResources[]
   ) => void
   titleText: string
+  isCooperation?: boolean
+  addNewSection?: (index?: number) => void
 }
 
 const CourseSectionsList: FC<CourseSectionsListProps> = ({
   items,
   setSectionsItems,
   handleSectionInputChange,
   handleSectionNonInputChange,
-  titleText
+  titleText,
+  isCooperation = false,
+  addNewSection = () => {}
```

Comment:
Guess you can leave it without default value and then call this function like:
addNewSection?.(currentSectionIndex)